app/Integrations/Base/WebhookPlugin.php

```
@@ -0,0 +1,139 @@
+            abort(401, 'Invalid webhook signature');
+        }
+        
+        // Process the webhook payload
+        $payload = $request->all();
+        $convertedData = $this->convertData($payload, $integration);
+        
+        // Create events, objects, and blocks
+        $this->createEventsFromWebhook($convertedData, $integration);
+    }
+    
+    protected function verifyWebhookSignature(Request $request, Integration $integration): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)